web/src/js/__tests__/ducks/ui.js

```diff
@@ -0,0 +1,34 @@
+jest.unmock("../../ducks/ui");
+
+import reducer, {setActiveMenu, SELECT_FLOW} from '../../ducks/ui';
+
+
+describe("ui reducer", () => {
+    it("should return the initial state", () => {
+        expect(reducer(undefined, {})).toEqual({ activeMenu: 'Start'})
+    }),
+    it("should return the state for view", () => {
+        expect(reducer(undefined, setActiveMenu('View'))).toEqual({ activeMenu: 'View'})
+    }),
+    it("should change the state to Start", () => {
```

Comment:
Let's be a bit more explicit about this: "should change the state to Start when deselecting a flow and we are currently on the flow tab"

(same below)